profile.html

```diff
@@ -22,18 +22,8 @@
   </div>
 </div>
 <script>
-  // Pagination controls
-  document.addEventListener("outboxUpdated", () => {
-    const outboxElement = document.querySelector("distributed-outbox");
-    const prevButton = document.getElementById("prevPage");
-    const nextButton = document.getElementById("nextPage");
-
-    if (prevButton) {
-      prevButton.addEventListener("click", () => outboxElement.prevPage());
-    }
-    if (nextButton) {
-      nextButton.addEventListener("click", () => outboxElement.nextPage());
-    }
+  document.addEventListener('usernameUpdated', (event) => {
+    document.title = `@${event.detail.username}`;
```

Comment:
Instead of replacing can we do a `+=` so we can still identify the tab as being part of the social reader?